configs/Phi-3-mini-4k-instruct/README.md

```diff
@@ -51,43 +51,38 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
-3. If you enable [Weights & Biases](https://wandb.ai/site), make sure you have a Weights & Biases account and [retrieve your API Key](https://wandb.ai/authorize)
 
 ### Provision Azure Resources
-To get started, you need provision the Azure Resource for remote fine-tuning. This can be done by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` from command palette.
+To get started, you need to provision the Azure Resource for remote fine-tuning. Do this by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning` from the command palette.
 
-You can monitor the progress of the provision via the link that is displayed in the output channel.
+Monitor the progress of the provision through the link displayed in the output channel.
 
-### Add ACA Secrets
-#### Weights & Biases API Key
-If you've enabled Weights & Biases in your project, it's crucial to set your Weights & Biases API Key for seamless access. Use the `AI Toolkit: Add Azure Container Apps Job secret for fine-tuning` command to input your Weights & Biases API Key. In this process, the secret name should be [`WANDB_API_KEY`](https://docs.wandb.ai/guides/track/environment-variables#optional-environment-variables) with your API Key as the secret value.
-
-#### [Optional] HuggingFace token
-If you're using private HuggingFace dataset, you should set your HuggingFace token as an environment variable to bypass the need for manual login on the Hugging Face Hub.
+### [Optional] Add Huggingface Token to the Azure Container App Secret
+If you're using private HuggingFace dataset, set your HuggingFace token as an environment variable to avoid the need for manual login on the Hugging Face Hub.
 You can do this using the `AI Toolkit: Add Azure Container Apps Job secret for fine-tuning command`. With this command, you can set the secret name as [`HF_TOKEN`](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#hftoken) and use your Hugging Face token as the secret value.
 
 ### Run Fine-tuning
 To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
 
 To view the system and console logs, you can visit the Azure portal using the link in the output panel (more steps at [View and Query Logs on Azure](https://aka.ms/ai-toolkit/remote-provision#view-and-query-logs-on-azure)). Alternatively, you can view the console logs directly in the VSCode output panel by running the command `AI Toolkit: Show the running fine-tuning job streaming logs`. 
-> **Note:** The job might take a few minutes to initiate. If there is already a running job, the current one may be queued to start later.
-    
+> **Note:** The job might be queued due to insufficient resources. If the log is not displayed after execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command, wait for a while and then execute the command again to re-connect to the streaming log.
```

Comment:
Add 'comma' after displayed and remove 'after', i.e. .....If the log is not displayed, execute the 'AI Toolkit.....

Reply:
Updated